tests/run-test.php

```diff
@@ -1,11 +0,0 @@
-<?php
-/**
- * This file is the main entry point for PHPUnit commands executed in continuous integration.
- * Since many of our tests rely on monkey patching techniques provided by uopz,
- * we must disable XDebug before firing up MediaWiki and running the actual tests.
- */
-if ( extension_loaded( 'xdebug' ) ) {
-	xdebug_disable();
```

Comment:
This is not actually necessary - it was added 7 years ago and only affects stack traces. We handle disabling XDebug Coverage - which would conflict with uopz - via command-line flags.